src/componets/DisplayDate.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { useEffect, useState} from "react"; 
+
+const DisplayDate = ()=> {
+
+    const [date, setDate] = useState()
+
+    useEffect (() =>{
+         getDate()
+      },[])
+
+    function getDate () {
+        const today = new Date();
+        const month = today.getMonth() + 1;
+        const year = today.getFullYear();
+        const date = today.getDate();
+        setDate(`${month}/${date}/${year}`)
+    }
+    return (
+        <div>
+            <p>Todays Date - {date}</p>
```

Comment:
QA  - According to requirements date should be bigger on the screen. Suggest using H1 or H2 tags